zerver/webhooks/gitlab/view.py

```diff
@@ -342,6 +343,23 @@ def get_pipeline_event_body(payload: WildValue, include_title: bool) -> str:
     )
 
 
+def get_release_event_body(payload: WildValue, include_title: bool) -> str:
+    action = payload["action"].tame(check_string)
+    name = payload["name"].tame(check_string)
+
+    if action == "delete":
+        body = f"Release {name} was deleted"
+    else:
+        url = payload["url"].tame(check_string)
+        body = f"Release [{name}]({url}) was {action}d"
```

Comment:
I think we can use the [RELEASE_MESSAGE_TEMPLATE](https://github.com/zulip/zulip/blob/main/zerver/lib/webhooks/git.py#L77) defined at `zerver.lib.webhooks.git`  as all required information is present in the payload.

For the delete event, it makes sense to create a template `RELEASE_MESSAGE_TEMPLATE_WITHOUT_URL` below the previous template as the URL in the payload 404s.

Reply:
I had that in mind at one point but then I noticed there is no detail about which user created the release from the gitlab event. But I could create a new template without the username maybe?

Reply:
I used the existing template (+ a new one) and set `user_name` to empty string and trimmed it to remove the space character in the beginning. Does it look alright? Or is a separate template without `user_name` better maybe?

Reply:
I was under the impression that the field `payload["commit"]["author"]["name"]` had the name of the user that made the release. But after testing some cases I can conclude that that is not the case.

The current sentence "Created release [v1.1](https://example.com/gitlab-org/release-webhook-example/-/releases/v1.1) for tag v1.1." doesn't sound good. 

Something like what you had done previously was better, eg. "Release [v1.1](https://example.com/gitlab-org/release-webhook-example/-/releases/v1.1) was created for tag v1.1.", creating a template for same makes sense.

```python
RELEASE_MESSAGE_TEMPLATE_WITHOUT_USER_NAME = "Release [{release_name}]({url}) for tag {tagname} was {action}."
RELEASE_MESSAGE_TEMPLATE_WITHOUT_USER_NAME_WITHOUT_URL = "Release {release_name} for tag {tagname} was {action}."
```

Reply:
Yeah, maybe gitlab will add that in the future.

It looks better. I pushed updates with your template. What do you think? :)